Formula/frieza.rb

```diff
@@ -0,0 +1,30 @@
+class Frieza < Formula
+    desc "A tool to cleanup your cloud resources"
+    homepage "https://github.com/outscale/frieza"
+    url "https://github.com/outscale-dev/frieza.git", :tag => "v0.6.0"
+    license "BSD-3-Clause"
+  
+    depends_on "curl"
+    depends_on "json-c"
+    depends_on "go" => :build
+    
+    def install
+      system "git", "submodule", "update", "--init"
+      system "make", "build"
+      bin.install "cmd/frieza/frieza"
+      completions = prefix/"contrib/completions"
+      bash_completion_file = "contrib/completions/bash/frieza-completion.bash"
+      zsh_completion_file = "#{completions}/zsh/frieza-completion.zsh"
```

Comment:
Sadly, I don't think frieza have a bash_completion file.
It would be a nice addition to frieza though.
In oapi-cli the file it copy is this one: https://github.com/outscale/oapi-cli/blob/master/oapi-cli-completion.bash
As oapi-cli-completion.bash is autogenerated, it might be a little hard to read, but it basically follows that: https://opensource.com/article/18/3/creating-bash-completion-script